src/datasources/asset-calibration/types.ts

```diff
@@ -141,5 +141,21 @@ export enum AssetFilterProperties {
 }
 
 export interface CalibrationForecastModel {
-  columns: FieldDTO[],
+  columns: FieldDTOWithDescriptor[],
+}
+
+export interface FieldDTOWithDescriptor extends FieldDTO {
+  columnDescriptors: ColumnDescriptor[]
+}
+
+export interface ColumnDescriptor {
+  value: string
+  type: string
```

Comment:
```suggestion
  type: ColumnDescriptorType
```

I think we could use the enum here so it's clear what to expect.